rust/libnewsboat-ffi/src/utils.rs

```diff
@@ -515,7 +515,7 @@ pub unsafe extern "C" fn rs_is_valid_podcast_type(mimetype: *const c_char) -> bo
 pub unsafe extern "C" fn rs_get_command_output(input: *const c_char) -> *mut c_char {
     abort_on_panic(|| {
         let rs_input = CStr::from_ptr(input);
-        let rs_input = rs_input.to_string_lossy();
+        let rs_input = rs_input.to_str().expect("input contained invalid UTF-8");
```

Comment:
We always add a comment explaining why `unwrap()` and `expect()` won't panic. Even though this `expect()` says what's wrong with the input, it doesn't explain why you think it won't happen. A lot of these comments are look-alikes; feel free to steal from `rs_run_command()` below.

Reply:
I've added a notice telling `This won't panic because all strings in Newsboat are in UTF-8` but I now found a way to still get it to panic.

Putting the following in a `urls` file:
```
https://newsboat.org/news.atom
exec:invalid�
```

created by putting the following in `urls.xxd` and running `xxd -r urls.xxd > urls` (0xff near the end is invalid):
```
00000000: 6874 7470 733a 2f2f 6e65 7773 626f 6174  https://newsboat
00000010: 2e6f 7267 2f6e 6577 732e 6174 6f6d 0a65  .org/news.atom.e
00000020: 7865 633a 696e 7661 6c69 64ff 0a         xec:invalid..
```

A Rust panic now happens when reloading the `exec:` feed.

Reply:
1. Is it useful if I open a separate bug for invalid characters in `urls` file? (similar to #723 but specific to the urls file)

2. Is it useful to delay merging this PR until the above issue is fixed?

Reply:
@dennisschagt, yes to both questions. Can you file the issue, please?

Reply:
Sure, done, see #844